apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/TracesResourceTest.java

```diff
@@ -349,6 +340,34 @@ void get__whenApiKeyIsPresent__thenReturnProperResponse(String apiKey, boolean e
             }
         }
 
+        @Test
+        @DisplayName("Get traces count on previous day for all workspaces, no Auth")
+        void tracesCountForWorkspace() {
```

Comment:
You should probably define a test subclass for the new endpoint and nest all the test cases there, even if you end up having a single test method.